test/Dapr.Actors.Test/Runtime/ActorRuntimeTests.cs

```diff
@@ -144,6 +145,51 @@ public async Task TestActorSettings()
 
             element = root.GetProperty("drainRebalancedActors");
             Assert.True(element.GetBoolean());
+
+            try {
```

Comment:
C# uses Allman style (braces on separate lines)

Reply:
Ok will update.

I am at the mercy of linters. And VS Code C# extension didn't catch this either.